resources/views/profile/delete-user-form.blade.php

```diff
@@ -23,41 +23,53 @@
             <x-slot name="description">
                 <div class="flex flex-col">
                     <div class="flex justify-center w-full">
-                        <x-ark-icon name="fortify-modal.delete-account" class="mt-8 mb-4 w-60 h-auto text-theme-primary-600 light-dark-icon"/>
+                        <x-ark-icon :name="$icon ?? 'fortify-modal.delete-account'" class="mt-8 mb-4 w-60 h-auto text-theme-primary-600 light-dark-icon"/>
                     </div>
 
                     @if($alert)
                         <div class="mt-4">
-                            <x-ark-alert type="info" class="py-8 px-4 mx-auto">
+                            <x-ark-alert :type="$alertType" class="py-4 mx-auto">
                                 {!! $alert !!}
                             </x-ark-alert>
                         </div>
                     @endif
 
+                    @if($showConfirmationMessage)
                     <div class="mt-4">
                         @lang('ui::forms.delete-user.confirmation')
                     </div>
+                    @endif
                 </div>
-                <form class="mt-8">
-                    <div class="space-y-2">
-                        <x-ark-input
-                            type="password"
-                            name="confirmedPassword"
-                            model="confirmedPassword"
-                            :label="trans('ui::forms.your_password')"
-                        />
-                    </div>
-                    <div class="mt-4">
-                        <x-ark-textarea
-                            name="feedback"
-                            model="feedback"
-                            :label="trans('ui::forms.feedback.label')"
-                            :placeholder="trans('ui::forms.feedback.placeholder')"
-                            :auxiliary-title="trans('ui::forms.optional')"
-                            rows="5"
-                        />
+
+                @if($confirmPassword)
+                <div class="space-y-2">
+                    <x-ark-input
+                        type="password"
+                        name="confirmedPassword"
+                        model="confirmedPassword"
+                        :label="trans('ui::forms.your_password')"
+                    />
+                </div>
```

Comment:
would it be possible to make it into a slot and have a default for the password but otherwise allow the user to determine what gets confirmed? will avoid issues in the future when we need a name or email or whichever thing the project uses

Reply:
I don't think slots are possible in livewire components?

Reply:
right, sad. Change it to name/display name in that case as deployer has no username